src/Quartz/Xml/XMLSchedulingDataProcessor.cs

```diff
@@ -405,6 +407,13 @@ protected virtual void ProcessInternal(string xml)
                 }
             }
 
+            // check for triggers with the same name
+            var duplicatedTriggers = triggerEntries.GroupBy(c => c.Item.name).Where(c => c.Count() > 1);
```

Comment:
I think the grouping should be done using name and group? Trigger's identity is based on both name and group.